ci-values.yaml

```diff
@@ -0,0 +1,99 @@
+# Default values for pdns.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+api:
+  enabled: yes
+  key: PowerDNSAPI
+
+webserver:
+  allowFrom: 0.0.0.0/0
+dnsupdate:
+  enabled: no
+
+image:
+  repository: hmcts/powerdns-auth
+  tag: 4.1.5r2-v0.0.1
+  pullPolicy: IfNotPresent
+
+replicaCount: 1
+
+service:
+  type: ClusterIP
+
+resources:
+  limits:
+   cpu: 100m
+   memory: 1024Mi
+  requests:
+   cpu: 100m
+   memory: 128Mi
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+environment:
+  AUTOCONF: postgres
+  PGSQL_USER: pdns
+  PGSQL_PASS: pdns
+  PGSQL_DB: postgres
+  PGSQL_HOST: chart-powerdns-ci-test-postgresql
+  PDNS_DEFAULT_SOA_NAME: root.platform.hmcts.net
+
+keyVault:
+  vaultName: infra-vault-nonprod
+  resourceGroup: cnp-core-infra
+  subscriptionId: 1c4f0704-a29e-403d-b719-b90c34ef14c9
+  tenantId: 531ff96d-0ae9-462a-8d2d-bec7c0b42082
+  secrets:
+    dbUsernameSecret: chart-pdns-db-username
+    dbPasswordSecret: chart-pdns-db-password
+
+ui:
+  enabled: true
+  replicaCount: 1
+  image:
```

Comment:
do you need all this config in the ci values file?
I would expect just anything that needs to be overridden here

Reply:
I had replicaCount in there because the default was 2.  I've now changed it